qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -207,6 +207,24 @@ def convert_frame_change(self, shift, instruction):
         }
         return self._qobj_model(**command_dict)
 
+    @bind_instruction(instructions.SetFrequency)
+    def convert_set_frequency(self, shift, instruction):
+        """ Return converted `SetFrequencyInstruction`.
+
+        Args:
+            shift (int): Offset time.
+            instruction (SetFrequency): set frequency instruction.
+        Returns:
+            dict: Dictionary of required parameters.
+        """
+        command_dict = {
+            'name': 'sf',
```

Comment:
I guess later we will have to distinguish this and shift freq? I think it is fine anyway

Reply:
Agreed, we will probably have to come up with a different short name for `ShiftFrequency` instruction (perhaps `sftf`?).